frontend/static/js/passfromprevious.js

```diff
@@ -1,87 +1,126 @@
-if (window.location.search.indexOf('query') == 1) {
-    var urlParams = new URLSearchParams(window.location.search)
-    var q = urlParams.get('query')
-    var r = urlParams.get('relevance')
+if (window.location.search.indexOf("query") == 1) {
+  var urlParams = new URLSearchParams(window.location.search);
+  var q = urlParams.get("query");
+  var r = urlParams.get("relevance");
 
-    // The data we are going to send in our request
-    let data_package = {
-        query: q,
-        relevance: r
-    };
+  // The data we are going to send in our request
+  let data_package = {
+    query: q,
+    relevance: r,
+  };
 
-        //filling up the result
-    var segment = $("#filler");
-    segment.html("");
-    $("#filler").hide().append('<div class="text-center"><div class="spinner-border text-primary" role="status"><span class="sr-only">Loading...</span></div></div>').fadeIn(1000);
+  //filling up the result
+  var segment = $("#filler");
+  segment.html("");
+  $("#filler")
+    .hide()
+    .append(
+      '<div class="text-center"><div class="spinner-border text-primary" role="status"><span class="sr-only">Loading...</span></div></div>'
+    )
+    .fadeIn(1000);
 
-    // var url = "https://schemes.sg/schemespredict";
-    var url = '/schemespredict'
+  // var url = "https://schemes.sg/schemespredict";
+  var url = "http://127.0.0.1:8000/schemespredict";
```

Comment:
call the frontend api
```suggestion
  var url = "/schemespredict";
```